api/endpoints/fake/it_automation.go

```diff
@@ -0,0 +1,50 @@
+package fake
```

Comment:
maybe instead of `fake`, use the more standard term stub?
```suggestion
package stub
```

Reply:
Until now, i've never seen any go project use the term `stub`, but most of the projects use `fake` (e.g. kubernetes). So i'd like to stick to `fake`.